.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: node_js
 node_js:
   - "6.10.3"
+install: npm install
```

Comment:
Yes it does. It a conventional thing I do every project as a fallback. But since might be redundant, I will take it out.